wordle-web/pages/leaderboard.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <v-container fluid fill-height justify-center>
+    <v-card width=80%>
+      <v-card-title>Leaderboard</v-card-title>
+      <v-col justify="center" dense no-gutters>
+        <v-row cols="9">
+          <v-col colspan="3">
+            <v-card-text>Name</v-card-text>
+          </v-col>
+          <v-col colspan="2">
+            <v-card-text>Games Played</v-card-text>
+          </v-col>
+          <v-col colspan="2">
+            <v-card-text>Average Attempts</v-card-text>
+          </v-col>
+          <v-col colspan="2">
+            <v-card-text>Average Seconds</v-card-text>
+          </v-col>
+        </v-row>
+        <v-card v-if="!getSuccessful" :loading="!getSuccessful">
+          <v-card-title>Loading...</v-card-title>
+        </v-card>
```

Comment:
Another way to do this:
https://vuetifyjs.com/en/components/progress-linear/

However, you probably don't want to rely only on if the call was successful, because the user will be stuck in a loading state if it fails.